Events/app/src/main/java/com/dti/cornell/events/TagAdapter.java

```diff
@@ -57,6 +70,10 @@ public int getItemCount()
 		return tags.size();
 	}
 
+	public Set<Integer> getSelectedTags() {
```

Comment:
Not sure why it's private if you're just making a getter for it (which allows anyone to mutate the set anyway)